Source/msg.cpp

```diff
@@ -2270,6 +2270,21 @@ size_t OnCheatExperience(const TCmd *pCmd, size_t pnum) // NOLINT(misc-unused-pa
 	return sizeof(*pCmd);
 }
 
+size_t OnCheatStats(const TCmd *pCmd, size_t pnum) // NOLINT(misc-unused-parameters)
+{
+#ifdef _DEBUG
+	if (gbBufferMsgs == 1)
+		SendPacket(pnum, pCmd, sizeof(*pCmd));
+	else {
+		ModifyPlrStr(Players[pnum], 255);
+		ModifyPlrMag(Players[pnum], 255);
+		ModifyPlrDex(Players[pnum], 255);
+		ModifyPlrVit(Players[pnum], 255);

```

Comment:
I don't think that `CMD_CHEAT_STATS` is necessary in this case, since these functions to modify player stats send their own network packets. You can just have the debug function call these directly.